ogn/parser/parse.py

```diff
@@ -6,16 +6,18 @@
 from ogn.parser.exceptions import AprsParseError, OgnParseError
 
 
-def parse_aprs(message, reference_date=None):
+def parse_aprs(message, reference_date=None, reference_time=None):
     if reference_date is None:
```

Comment:
No, you're right. I can't find any case in which calling `parse_aprs` with `date=None` and `time=some_time` makes sense, so the exact behavior of the function in this case could be ignored.